package.json

```diff
@@ -19,7 +24,8 @@
     "url": "https://github.com/elixir-lang/tree-sitter-elixir.git"
   },
   "scripts": {
-    "test": "tree-sitter test",
+    "build": "npx tree-sitter-cli generate --no-bindings",
```

Comment:
I changed this to make the scripts less reliant on the system state. Running `npx tree-sitter-cli` instead of `tree-sitter` will ensure the version specified in the `devDependencies` is used.